test/functions/socialFunctionsTest.js

```diff
@@ -158,3 +158,62 @@ describe('socialVolume_', () => {
     expect(() => san.socialVolume_(slug, from, to, 'TELEGRAM_CHATS_OVERVIEW')).to.throw('No data')
   })
 })
+
+describe('news_', () => {
+  it('returns news for a slug', () => {
+    sandbox.stub(san.ApiClient_.prototype, 'fetchNews').returns(
+      [
+        {
+          datetime: '2019-07-06T23:06:36Z',
+          title: 'Santiment Network Token Reaches 1-Day Volume of $17,181.00 (SAN) - Mayfield Recorder',
+          description: 'Santiment Network Token Reaches 1-Day Volume of $17,181.00 (SAN)  Mayfield RecorderSantiment Network Token (CURRENCY:SAN) traded 1.9% higher against the US dollar during the 1 day period ending at 19:00 PM ET on July 6th. Santiment ...',
```

Comment:
Shorten these fields to fit in 120 chars per row limit. The data is fake so you can add `...` at the end.  
Example : `'Santiment Network Token Reaches 1-Day Volume of $17,181.00 (SAN)...`